apis/vshn/v1/dbaas_vshn_postgresql.go

```diff
@@ -64,6 +67,12 @@ type VSHNPostgreSQLServiceSpec struct {
 	PostgreSQLSettings runtime.RawExtension `json:"pgSettings,omitempty"`
 }
 
+// VSHNDBaaSSchedulingSpec contains settings to control the scheduling of an instance.
+type VSHNDBaaSSchedulingSpec struct {
+	// NodeSelector is a selector which must match a node’s labels for the pod to be scheduled on that node
+	NodeSelector runtime.RawExtension `json:"nodeSelector,omitempty"`
```

Comment:
```suggestion
	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
```

I didn't see that last time. I'd suggest to be more restrictive and use the same type as upstream: https://github.com/kubernetes/api/blob/master/core/v1/types.go#L3191-L3197